sample/src/main/java/com/decathlon/compose/sample/screens/Cards.kt

```diff
@@ -0,0 +1,267 @@
+package com.decathlon.compose.sample.screens
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.ScrollState
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement.Bottom
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.decathlon.compose.sample.R
+import com.decathlon.compose.sample.components.SampleScaffold
+import com.decathlon.vitamin.compose.buttons.VitaminButtons
+import com.decathlon.vitamin.compose.buttons.VitaminIconButtons
+import com.decathlon.vitamin.compose.cards.VitaminCards
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+import com.decathlon.vitamin.compose.foundation.vtmnTypography
+
+object Cards : Screen {
+    override val name: String
+        get() = "Cards"
+
+    override val navigationKey: String
+        get() = "cards"
+
+    @SuppressWarnings("LongMethod")
```

Comment:
Maybe create a private composable function for each card sample & remove this `SuppressWarnings`

Reply:
Done